_data/icons/spectre.json

```diff
@@ -0,0 +1,8 @@
+[
+    {
+      "url": "ipfs://QmPbWoQ6gdfwkeoqu6mC3BR97bNP6s3hcfhnsRKiNCAGYH",
```

Comment:
where is this pinned?

Reply:
@ligi @Anastasia1510 
HarryxDD has left our team with his github account, so we can't update the icon image. 
I made another pull request by my github account (updated ipfs link of the icon image) here:
https://github.com/ethereum-lists/chains/pull/1566

Please close this pull request and move to new one.